docs/modules/ROOT/pages/vshn-managed/keycloak/create.adoc

```diff
@@ -0,0 +1,54 @@
+= Create a Keycloak service
+
+Apply the following object on your namespace, as specified by its YAML description.
+
+.Example to create a Keycloak instance
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNKeycloak
+metadata:
+  name: keycloak-app1-prod # <1>
+  namespace: prod-app # <2>
+spec:
+  parameters:
+    service:
+      version: "23" # <3>
+      postgreSQLParameters: {} # <4>
+    size: # <5>
+      plan: standard-2
+  writeConnectionSecretToRef:
+    name: keycloack-creds # <6>
+----
+<1> Instance name
+<2> The namespace where the object will be created
+<3> Keycloak version
+<4> PostgreSQL database parameters. See xref:vshn-managed/postgresql/index.adoc[PostgreSQL by VSHN]
+<5> Size of the Keycloak instance. See xref:vshn-managed/keycloak/plans.adoc[Plans and Sizing] for more information.
+<6> Secret where the connection details are provisioned. This secret shouldn't exist before creation.
+
+NOTE: To get more information about all available configuration options, please see the xref:references/crds.adoc#k8s-api-github-com-vshn-component-appcat-apis-vshn-v1-vshnmkeycloak[API Reference]
+
+== Inspect your new Keycloak service
+
+[source,bash]
+----
+$  oc get vshnkeycloak.vshn.appcat.vshn.io
+NAME                SYNCED   READY   CONNECTION-SECRET   AGE
+keycloak-app1-prod   True     True    keycloak-creds       3m32s
+----
+
+When you see `True` in the READY column, it means the instance is provisioned and ready to use.
```

Comment:
> and ready to use.

That's unfortunately not true, Keycloak takes a loooong time to start, especially the first time.

`ready: true` only means that all objects and the helm chart got applied.